core/trino-main/src/main/java/io/trino/sql/planner/planprinter/PlanPrinter.java

```diff
@@ -932,15 +958,17 @@ private Void visitScanFilterAndProjectInfo(
                 }
             }
 
+            Collection<DynamicFilters.Descriptor> dynamicFilters = ImmutableList.of();
             if (filterNode.isPresent()) {
                 operatorName += "Filter";
                 formatString += "filterPredicate = %s, ";
                 Expression predicate = filterNode.get().getPredicate();
                 DynamicFilters.ExtractResult extractResult = extractDynamicFilters(predicate);
                 arguments.add(unresolveFunctions(combineConjunctsWithDuplicates(extractResult.getStaticConjuncts())));
                 if (!extractResult.getDynamicConjuncts().isEmpty()) {
-                    formatString += "dynamicFilter = %s, ";
```

Comment:
that seems like a bug. Could you extract to separate commit?

Reply:
There's no bug, I'm just collecting `dynamicFilters` for later call to printDynamicFilterDomainStats

Reply:
you changed format from `dynamicFilter = %s` to `dynamicFilters = %s`

Reply:
ok, made the addition of `s` a separate commit